### PR TITLE
feature(OSS-663): notify users about new version

### PR DESCRIPTION
**Pull request by fireridlle**
[Notify users via command line when driver has been updated](https://faunadb.atlassian.net/browse/OSS-663)

image

### CODE REVIEW

File: faunadb/__init__.py [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+
+
+response = requests.get('https://pypi.org/pypi/faunadb/json')
+latest_version = response.json().get('info').get('version')
+latest_version_arr = latest_version.split(".")
+current_version = __version__.split(".")
+is_new_version_available = False
+
+for i in range(len(current_version)):
+  if latest_version_arr[i] > current_version[i]:
```
Comment (parkhomenko) [inline faunadb/__init__.py:18]: I understand that this situation is not really possible, but what if `current_version` is 5.0.0 and `latest_version` is 4.2.0. I think the `is_new_version_available` returns true.

File: faunadb/__init__.py [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+    is_new_version_available = True
+
+
+if True: #is_new_version_available:
```
Comment (parkhomenko) [inline faunadb/__init__.py:22]: Do you need to return the commented version?

**Review by parkhomenko — CHANGES_REQUESTED** [review]
Can we put the code starting from line 12 and add 1 or 2 unit tests?
I mean you can create a function that accepts 2 params: current_version and latest_version and returns the message that needs to be rendered in case the latest_version is newer than the currently installed.

**Review by parkhomenko — APPROVED** [review]

**Review by vadimLF — APPROVED** [review]
LGTM

**Pull request merged**